nilearn/image/tests/test_resampling.py

```diff
@@ -163,8 +163,8 @@ def test_resampling_with_affine():
     """
     rng = np.random.RandomState(42)
 
-    data_4d = rng.randint(4, size=(1, 4, 4, 3))
-    data_3d = rng.randint(4, size=(1, 4, 4))
+    data_4d = rng.randint(4, size=(1, 4, 4, 3), dtype=np.int32)
+    data_3d = rng.randint(4, size=(1, 4, 4), dtype=np.int32)
```

Comment:
Why not int32 ?

Reply:
(same comments for the lines below)

Reply:
do you mean why not `dtype="int32"` instead of `dtype=np.int32`? I don't have a preference, I can change to the string version

Reply:
I had the impression that it was recommended to use the string version.